src/main.tsx

```diff
@@ -1,8 +1,9 @@
-import { StrictMode } from 'react';
-import { createRoot } from 'react-dom/client';
-import App from './App';
-import './index.css';
```

Comment:
Importing CSS is not supported in deno typecheck. It was the big blocker 😢 

https://github.com/denoland/deno/issues/11961